lib/cog_api/http/relay_groups.ex

```diff
@@ -39,16 +39,24 @@ defmodule CogApi.HTTP.RelayGroups do
   end
 
   def add_relay(%{name: name}, %{relay: relay_name}, %Endpoint{}=endpoint) do
```

Comment:
So I ended up refactoring the add_bundle bit in my branch. The reason I mention it is because it is super similar to this bit. The `add_relay` and `remove_relay` functions actually use the exact same logic as one another. The only difference is when they pass off to `update_memberships` they pass an atom to either `:add` or `:remove` the relay. Like I said, this is super similar to `add_bundle` and `remove_bundle`. I ended up refactoring and just merging both functions into `update_assignments`. Curious what your thoughts are on that? https://github.com/operable/cog-api-client/pull/89/files#diff-e6847729ab31281c5ad0f078cc6a8f23R70